backend/contracts/Quiz.sol

```diff
@@ -36,18 +47,19 @@ contract Quiz {
     address private _owner;
     // Encryption key for encrypting payout certificates.
     bytes32 private _key;
-    // Quiz ID for generating NFT images
-    uint32 private _quizID;
     // Base64 svg image for this quiz instance
     string private _svgImage;
     // NFT contract address
     address private _nftAddress;
+    // reward contract image counter
+    uint256 private _imageCounter;
```

Comment:
```suggestion
    // Hash of the reward NFT token URI
    bytes32 private _tokenURIHash;
```

Reply:
Removed _imageCounter and added _jsonHash attribute.